packages/components-js/src/pages/flyout.html

```diff
@@ -0,0 +1,920 @@
+<style>
+  body {
+    overflow: auto !important;
+    background: deeppink;
+  }
+
+  .playground {
+    height: 600px;
+    padding: 0;
+    background: linear-gradient(
+      90deg,
+      rgba(20, 84, 255, 0) 0%,
+      rgba(20, 84, 255, 1) 50%,
+      rgba(20, 84, 255, 0) 100%
+    ) !important;
+    transform: translate3d(0, 0, 0);
+  }
+  .playground {
+    margin: 16px 0;
+  }
+
+  @media only screen and (min-width: 760px) {
+    #app,
+    :host {
+      display: grid;
+      grid-template-columns: repeat(2, 50%);
+    }
+  }
+</style>
+
+<div class="playground light auto-layout" title="should show flyout">
+  <p-flyout open="true">Some Content: should show flyout</p-flyout>
+</div>
+
+<div class="playground dark auto-layout" title="should show flyout on dark theme">
+  <p-flyout open="true" theme="dark">Some Content: should show flyout</p-flyout>
+</div>
+
+<div class="playground light auto-layout" title="should show flyout with prefixed component initialization">
+  <my-prefix-p-flyout open="true">
```

Comment:
Hmm, why do we do the prefixing coverage part here? That's always handled on the overview page.

Reply:
Modal also had it on that page thats why i added it here as well